### ISSUE TITLE
Check into Google Ad Grants

### ISSUE THREAD

**Issue by laryn** [backdrop-ops/backdropcms.org#542]
Pasted from Gitter:

>  I was curious if anyone had looked into whether Backdrop (as a SFC entity) qualified for Google Ad Grants. If so, Google gives up to $10k / month in free Google Adwords.

> But you might need to be a stand alone 501c3. Not certain.

> Digging a little deeper, I think this is the short list of eligibility:

>> Organizations must be recognized by the IRS as tax-exempt 501(c)(3) organizations formed for charitable, educational, religious, literary, scientific, or other tax-exempt purposes. Fiscally sponsored organizations are not eligible for Google for Nonprofits. Organizations, such as churches, that are automatically considered tax-exempt under group exemption must still obtain recognition of 501(c)(3) tax exemption from the IRS to qualify for the Google for Nonprofits program.
>> Organizations must be validated by TechSoup.

https://support.google.com/nonprofits/answer/3215869?hl=en&ref_topic=3247288

>> Fiscally sponsored organizations that do not have their own 501(c)(3) status and are not covered by group exemption are not eligible for the Google for Nonprofits program.

Is that what we are?
Maybe this is something our SFC contact can answer.

>> What is a group exemption letter?
The IRS sometimes recognizes a group of or-ganizations as tax-exempt if they are affiliated with a central organization. This avoids the need for each of the organizations to apply for exemption individually. A group exemp-tion letter has the same effect as an individualexemption letter except that it applies to more than one organization.

**Comment by jenlampton** [comment]
> Fiscally sponsored organizations that do not have their own 501(c)(3) status and are not covered by group exemption are not eligible for the Google for Nonprofits program.

This is what we are, but I don't know if we are covered by a group exemption or not. We should ask the SFC.

**Comment by laryn** [comment]
It looks like they've stripped a lot of the eligibility language off the site and now it just says you need to be verified by TechSoup. Do I have permission to try to add Backdrop as an organization on TechSoup? @jenlampton

**Comment by jenlampton** [comment]
yes please :)

**Comment by docwilmot** [comment]
Any progress on this @laryn ?

**Comment by jenlampton** [comment]
We're still waiting on a legal review of the TOS from the SFC.

**Comment by laryn** [comment]
@jenlampton @docwilmot  I'm not getting much response to my emails from our contact anymore.

**Comment by jenlampton** [comment]
@laryn I think we should start the application process with google ad grants. We don't know how long it will take, it could be a while. Let's tell SFC that's what we're doing, and if their legal team does decide it's a problem we can back out of the ad grants program when we find out.

**Comment by laryn** [comment]
@jenlampton I tried to set up an organizational account at TechSoup. I didn't have an address for Backdrop CMS so I just used a `c / o` along with the office address of my org for now. I used the `info@` email address and a phone number that may be yours (plucked from an old email). 

TechSoup will verify that we qualify or not and will get back to us, most likely at the `info@` email address. Does that go to you?

**Comment by jenlampton** [comment]
@laryn that sounds great. Yes, I can see emails sent to `info@` I'll let you know if I hear anything :)

**Comment by laryn** [comment]
I have sent (another) followup. I heard the following in May 2021 as to whether Backdrop could access free Google Ad Grants from underneath the SFC umbrella and I've followed up once since then but have not received a reply:

> I forgot to mention to you that I'm looking into it. It looks like we have some experience in the past with the program but it's changing nature means we need to reevaluate the terms and uses.
 
> I'm hopeful that we can get you under the SFC umbrella if our legal team approves it and will keep you updated when I hear back from them or learn anything new.

**Comment by cellear** [comment]
Bump :)

**Comment by laryn** [comment]
I have had no luck getting a response and may have lost the previous thread completely as I've moved jobs and lost the inbox I was writing from. @jenlampton do you have a copy of the latest email in that thread at the `info@` address?

**Comment by jenlampton** [comment]
@laryn  yes, I will forward you a copy. We have been having fortnightly meetings with the SFC and I can add this to our agenda for Dec 6th. Laryn would you like to join that meeting and meet our current SFC contacts F2F? I'll add you to the invite.

**Comment by jenlampton** [comment]
@laryn oh, I'll need your new address! Will PM on Zulip.

**Comment by jenlampton** [comment]
Email forwarded, I also sent you addresses for Pono and Karen.
